tests/angular-localForage.js

```diff
@@ -183,20 +183,41 @@ describe('Module: LocalForageModule', function() {
 
   it('setItem and getItem should work', function(done) {
     var interval = triggerDigests();
+    const timestamp = new Date();
```

Comment:
No ES2015 in this codebase for now. I plan on doing a good sprint on this where at least the tests get updated, but until we change it all, let's stick with `var`.